mkimg.lima.sh

```diff
@@ -6,11 +6,15 @@ profile_lima() {
 		Slimmed down kernel.
 		Optimized for virtual systems.
 		Configured for lima."
-	arch="aarch64 x86 x86_64"
+	arch="aarch64 armv7 x86 x86_64"
 	initfs_cmdline="modules=loop,squashfs,sd-mod,usb-storage"
 	kernel_addons=
 	kernel_flavors="virt"
-	kernel_cmdline="console=tty0 console=ttyS0,115200"
+	case "$ARCH" in
+		arm*|aarch64)
+			kernel_cmdline="console=tty0 console=ttyAMA0"
```

Comment:
Why don't we need to set the console on x86_64?

Reply:
It was changed upstream, I think it now uses the default ?


https://github.com/alpinelinux/aports/commit/4ad0b08bfa38cf664469c28a0e35492e0de56e86
https://github.com/alpinelinux/aports/commit/f81ba92e56993a63a92a5f4de83da79734ed2540